commons/connector/altoida/AltoidaLog.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaLog",
+  "doc": "Logs of actions taken.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeRecieved", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "log_actions", "type": "string", "doc": "Log of actions taken during test."},
```

Comment:
Is this multiple actions, if it is one, it can just be `action`. If there is a defined subset that we got in our data, it would be good to document at least those.